tools/haskell-translator/lhs_pars.py

```diff
@@ -1407,10 +1430,10 @@ def get_type_map(string):
         return (header, [])
     actual_map = bits[1].strip()
     if not (actual_map.startswith('{') and actual_map.endswith('}')):
-        print('Error in ' + filename)
-        print('Expected "A { blah :: blah etc }"')
-        print('However { and } not found correctly')
-        print('When parsing %s' % string)
+        error(f'in {filename}\n'
+              'Expected "A { blah :: blah etc }"\n'
+              'However { and } not found correctly\n'
+              f'When parsing {string}')
```

Comment:
Very minor, but you're currently not consistent in whether you make all lines of multiline strings f-strings, or just the lines that have expressions.

Reply:
Ah, right. Based on me just learning about f-strings and reading up on them, I think you would use double braces in that case.

Reply:
Yes, you could escape them, but I find that less readable than just not using an f-string when it's not necessary.